### PR TITLE
 sign_up request to db

### PR DESCRIPTION
**Pull request by BrilStrawhat**
Functions create row with login and passwd if login uniq and assign uniq id.

### CODE REVIEW

**Comment by BrilStrawhat** [comment]
@Hvvang Could you please help me correct fix conflict?

File: server/src/server.c [hunk 1]
```diff
@@ -182,7 +122,7 @@ int main(int argc, char **argv) {
     if ((rc = sqlite3_exec(*db, "SELECT User_id FROM users_credential LIMIT 1;",
                     NULL, NULL, &errmsg)) != SQLITE_OK) {
         if ((rc = sqlite3_exec(*db, "CREATE TABLE users_credential ( \
-                            user_id INTEGER NOT NULL, \
+                            user_id INTEGER, \
                             login VARCHAR(50) NOT NULL, \
                             passwd_hash VARCHAR(32) NOT NULL, \
                             PRIMARY KEY(user_id));", NULL, NULL, &errmsg)) != SQLITE_OK) {
```
Comment (Hvvang) [inline server/src/server.c:161]: Rewrite it following by the Auditor rule.

File: server/src/server.c [hunk 2]
```diff
@@ -182,7 +122,7 @@ int main(int argc, char **argv) {
     if ((rc = sqlite3_exec(*db, "SELECT User_id FROM users_credential LIMIT 1;",
                     NULL, NULL, &errmsg)) != SQLITE_OK) {
         if ((rc = sqlite3_exec(*db, "CREATE TABLE users_credential ( \
-                            user_id INTEGER NOT NULL, \
+                            user_id INTEGER, \
                             login VARCHAR(50) NOT NULL, \
```
Comment (Hvvang) [inline server/src/server.c:159]: Did u set user_id to be uniq one?

**Review by Hvvang — APPROVED** [review]
Please, make `db_func` a single file.

**Review by Hvvang — DISMISSED** [review]
Everything ok

**Review by Hvvang — APPROVED** [review]

**Comment by Hvvang** [comment]
Sign In/Up release

**Comment by BrilStrawhat** [comment]
Why did you merge it?

**Pull request merged**